source/beamline/IDEAS/IDEAS13ElementGeDetector.h

```diff
@@ -47,7 +47,15 @@ class IDEAS13ElementGeDetector : public AMXRFDetector
 	/// The vortex detectors support elapsed time.
 	virtual bool supportsElapsedTime() const { return true; }
 
+	double peakingTime() const { return peakingTimeControl_->value(); }
```

Comment:
Remember that functions should always be commented.  Even if it is just as simple as "Returns the peaking time for the detector."  It's always good practice to do it as you go because you often-times don't make it back to comment it when you're done.